lib/rules/balanced-braces.js

```diff
@@ -0,0 +1,61 @@
+'use strict';
+
+const types = require('../utils/types');
+const ember = require('../utils/ember');
+
+//------------------------------------------------------------------------------
+// General rule -  Braces has to be balanced.
+//------------------------------------------------------------------------------
+
+const ERROR_MESSAGE = 'Unbalanced braces';
+
+module.exports = {
+  meta: {
+    type: 'problem',
+    docs: {
+      description: 'require that braces to be balanced.',
```

Comment:
Description should mention that this is referring to computed property dependent keys. Otherwise it's unclear what braces this is referring to.